demos/src/icons.json

```diff
@@ -0,0 +1,34 @@
+{
+  "icons": [
+    {
```

Comment:
Should this be tab-indented? Think it's only in `package.json` that we use two spaces

Reply:
:+1: